src/lib/Libwin/passwd.c

```diff
@@ -2533,7 +2534,7 @@ default_local_homedir(char *username, HANDLE usertoken, int ret_profile_path)
 		} else {
 			userlogin = LogonUserNoPass(username);
 			if (userlogin == 0) {
```

Comment:
Do you not want to print the last error code? Since you are moving from log_errf() which could print the last error to log_evebtf() which cannot, do you not want to mention the lasteerror explicitly?

Reply:
I don't think here GetLastError() will show the real reason of any error, as LogonUserNoPass() is PBS API and the corresponding function have proper error handling.